src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -0,0 +1,94 @@
+const moment               = require('moment');
+const DigitTicker          = require('./digit_ticker');
+const ViewPopupUI          = require('../user/view_popup/view_popup.ui');
+const showLocalTimeOnHover = require('../../base/clock').showLocalTimeOnHover;
+const BinarySocket         = require('../../base/socket');
+const localize             = require('../../../_common/localize').localize;
+const getPropertyValue     = require('../../../_common/utility').getPropertyValue;
+
+const DigitDisplay = (() => {
+    let $container,
+        contract,
+        tick_count;
```

Comment:
perhaps can be renamed to current_tick_count as well.

Reply:
This is total tick count. I updated variable names to be more clear.